docs/08_crosscutting_concepts/08_crosscutting_concepts.md

```diff
@@ -2,4 +2,212 @@
+$colorLink: #c6264e;
+$colorLinkHover: #3d3d3c;
+$colorLinkActive: #95223f;
+$colorRatingGood: #b1ffa5;
+$colorRatingBad: #ff728b;
+$colorRatingMedian: #ffd76f;
+
+:export {
+    colorPrimary: $colorPrimary;
+    colorSecondary: $colorSecondary;
+    colorTertiary: $colorTertiary;
+    colorTextHeading: $colorTextHeading;
+    colorBgContainer: $colorBgContainer;
+    colorBgContainerHighlight: $colorBgContainerHighlight;
+    colorLink: $colorLink;
+    colorLinkHover: $colorLinkHover;
+    colorLinkActive: $colorLinkActive;
+    colorRatingGood: $colorRatingGood;
+    colorRatingBad: $colorRatingBad;
+    colorRatingMedian: $colorRatingMedian;
+}
+```
+
+#### Color Schema
+We developed the color scheme according to the HFTM corporate design.
+
+| Farbvariable   | Hex-Wert  |
+| -------------- | --------- | 
+| Primary        | `#C6264E` | 
+| On Primary     | `#FFFFFF` | 
+| Secondary      | `#95223F` | 
+| On Secondary   | `#FFFFFF` | 
+| Background     | `#E4E4E4` | 
+| On Background  | `#3D3D3C` |
+| Neutral 300    | `#f2f2f2` | 
+| Neutral 200    | `#f9f9f9` | 
+
+
+| primary & secondary   | background & neutrals  |
+| -------------- | --------- | 
+| ![image](.\primary-secondary-color.png)| ![image](.\background-color.png) | 
+
+
+
+#### SCSS Styles in React Components
+Typical structure
+
+components: <br>
+├───SelectWithTitle.tsx <br>
+├───SelectWithTitle.module.scss
+
+Here you can see the structure of a typical React component in our project. To import and use the styles correctly, please note the following points:
+- The SCSS files should have `.module.scss` in the name (e.g., `SelectWithTitle.module.scss`). This is the only way for autocomplete to work correctly and enable local scope for CSS class names.
+
+- Import the styles with the following syntax:
+```javascript
+import styles from './SelectWithTitle.module.scss';
+```
+
+- To access SCSS elements, use the styles object:
+```javascript
+<div className={styles.selectWithTitleContainer}>
+  {/* Here your content*/}
+</div>
+```
+
+- When defining the SCSS elements, always use **camelCase**. This makes it easier to access via the styles object in React. For example:
+
+This approach limits styles to the respective component, which avoids naming conflicts.
+
+### 8.4 Validation
+To validate the data in the frontend, we rely on the functions of Antd Forms. It is possible to use predefined functions such as required or to use CustomValidators.
+
+Example:
+```TS
+const notNullValidator: Rule = {
+    required: true,
+    message: 'Input is required != null',
+};
+```
+
+### 8.5 Error handling
+We also rely on the Ant components to display error messages visually.
+
+We can see relevant examples here:
+https://ant.design/components/form#rule
+
+### 8.7 Testability
+Due to the amount of work involved, we will initially avoid unit tests and rely on manual system tests.
+
+
+To test the different roles you can use the follwing script:
+```PowerShell
```

Comment:
Not sure if this is on the right place here, maybe we could create the script in the report somewhere and just link it here?

Reply:
Removed and added the link to the existing wiki for now.